blobfuse/blobfuse.cpp

```diff
@@ -762,24 +797,35 @@ bool is_directory_empty(const char *tmpDir) {
 }
 
 
-int 
-
-read_and_set_arguments(int argc, char *argv[], struct fuse_args *args)
+int read_and_set_arguments(int argc, char *argv[], struct fuse_args *args)
 {
     // FUSE has a standard method of argument parsing, here we just follow the pattern.
     *args = FUSE_ARGS_INIT(argc, argv);
 
     // Check for existence of allow_other flag and change the default permissions based on that
     config_options.defaultPermission = 0770;
     config_options.readOnlyMount = false;
+    config_options.entryTimeout = 240;
```

Comment:
any reason this default is added here. datasetfuse has its own defaults optimized for performance as per what they think